diff --git a/broker/setup.py b/broker/setup.py
index b150830..8e330f4 100644
--- a/broker/setup.py
+++ b/broker/setup.py
@@ -25,12 +25,8 @@ setup(
     include_package_data=True,
     zip_safe=False,
     setup_requires=[
-        'cffi>=1.4.1',
     ],
     install_requires=[
-        'netfilter>=0.6.2',
-        'six>=1.10.0',
-        'cffi>=1.4.1',
     ],
     extras_require={},
     cffi_modules=[
